.gitignore

```diff
@@ -62,3 +62,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+openassessment/xblock/static/css/openassessment-ltr.css.map
```

Comment:
Hmm, they're green in this diff, which means they were added with this work or are based on work that is different from master. Feel free to remove them if they are not needed.